sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -162,10 +175,6 @@ function getRequestBody(body?: unknown, contentType: string = ""): RequestBody {
     return { body };
   }
 
-  if (!contentType && typeof body === "string") {
-    return { body };
-  }
-
```

Comment:
we will never went into the condition here now, so removed it.

Reply:
I think this is what is causing the added quotes

Reply:
oh actually, we will never fall into this condition as of the change here https://github.com/Azure/azure-sdk-for-js/pull/27343/files#r1351774925

Reply:
The added quotes should have been resolved as of the default content-type has been changed into text/plain if it's non-json string.